src/ui/icons/icon-success.svelte

```diff
@@ -0,0 +1,15 @@
+<script>
+  import Icon from "../../assets/success.svg?raw"; // Load SVG as a raw string
+</script>
+
+<div style="color:var(--rc-color-accent);" class="rcb-ui-asset-icon">
```

Comment:
Here I modified the svg file to use `fill="currentColor"`.

In this way the css style can change the color of the resulting icon.
That's how fontawesome svg icons work as well.